src/plugins/telemetry/server/telemetry_collection/get_local_stats.test.ts

```diff
@@ -198,6 +198,19 @@ describe('get_local_stats', () => {
       expect(result.stack_stats).toEqual({ kibana: undefined, data: undefined });
     });
 
+    it('fallbacks to Kibana version if ES does not respond with it', () => {
+      const { version: _, ...clusterInfoWithoutVersion } = clusterInfo;
+      const result = handleLocalStats(
+        clusterInfoWithoutVersion as estypes.InfoResponse,
+        clusterStatsWithNodesUsage,
+        void 0,
+        void 0,
+        context
+      );
+
+      expect(result.version).toEqual('8.0.0');
```

Comment:
NIT: `toEqual(context.version)`

Reply:
++ I noticed that all other tests in this file checked against actual strings instead of the var containing the strings. I thought we preferred being explicit about it.